bottlecap/src/main.rs

```diff
@@ -123,6 +125,7 @@ fn main() -> Result<()> {
 
     let r = register().map_err(|e| Error::new(std::io::ErrorKind::InvalidData, e.to_string()))?;
 
+    let mut logs_agent = LogsAgent::run("TODO"); // todo how to add function arn?
```

Comment:
I think you can build the arn from:
- env vars
- accountID from Lambda-Extension-Accept-Feature (see: https://docs.aws.amazon.com/lambda/latest/dg/runtimes-extensions-api.html#extensions-registration-api-a)

Reply:
Oh, nice perfect, thanks for this